docs/components/concepts/processes.md

```diff
@@ -4,82 +4,60 @@ title: "Processes"
 description: "Processes are flowchart-like blueprints that define the orchestration of tasks."
 ---
 
-Processes are flowchart-like blueprints that define the orchestration of **tasks**.
+A process is a defined sequence of distinct steps or tasks representing your business logic. Examples of a process could be an e-commerce shopping experience, or onboarding a new employee.
 
-For example, with Camunda you can [orchestrate human tasks](../../guides/getting-started-orchestrate-human-tasks.md).
+At large, process orchestration is a technology that coordinates the various moving parts (or endpoints) of a business process, and sometimes even ties multiple processes together. Process orchestration helps you work with the people, systems, and devices you already have—while achieving goals around end-to-end process automation.
```

Comment:
Is this aligned with Marketing? Part of why I ask is the examples and inclusion of "devices." Our current product experience definitely includes people and systems, but I'm not so sure about devices.

Reply:
Devices was pulled from our web copy, but I can easily remove it if we just want to be safe.

Reply:
No need to remove it, but thanks for the context!